tests/test_circulationapi.py

```diff
@@ -898,27 +898,33 @@ def yes_we_can(*args, **kwargs):
         result = try_to_fulfill()
         eq_(fulfillment, result)
 
-    def test_revoke_loan_sends_analytics_event(self):
+    def test_revoke_loan(self):
+        self.patron.last_loan_activity_sync = datetime.utcnow()
```

Comment:
Using `utcnow()` here could cause the test to be ineffective if, in the future, `loan_activity_max_age` is set to a very small positive value or zero. In that case, the cache might expire before `revoke_loan` "invalidates" it, in which case we wouldn't know who was responsible for the invalidation.